docker/geth/scripts/node.sh

```diff
@@ -56,7 +56,7 @@ else
     echo "Starting geth VALIDATOR with BOOTNODE enabled on port '$BOOTNODE_PORT' with '/boot.key'"
     mkdir -p $DATA_DIR/keystore
     cp /keystore/$etherbase.json $DATA_DIR/keystore
-    geth --datadir $DATA_DIR \
+    exec geth --datadir $DATA_DIR \
```

Comment:
@Filter94 just to confirm, are all these changes for ensuring the geth command would run in the same shell process? such that sigterm would propagate properly to the process?

Reply:
I think your explanation is accurate. I did it by [this](https://cloud.theodo.com/en/blog/docker-processes-container) guide